OtripleS.Web.Api.Tests.Unit/Services/TeacherAttachments/TeacherAttachmentServiceTests.Exceptions.Remove.cs

```diff
@@ -0,0 +1,193 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.TeacherAttachments;
+using OtripleS.Web.Api.Models.TeacherAttachments.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.TeacherAttachments
+{
+    public partial class TeacherAttachmentServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnRemoveWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            var randomAttachmentId = Guid.NewGuid();
+            var randomTeacherId = Guid.NewGuid();
```

Comment:
Since we really don't care about where the id is coming from, in the case where the input is just a fill-in to get the process going, you don't need to define them on two steps. just do this:

```csharp
Guid someStudentId = Guid.NewGuid();

```

Reply:
Do the same for the rest